borderland-tool/borderland_tool/lottery/__init__.py

```diff
@@ -133,7 +133,7 @@ def load_orders_and_vouchers(self):
 
     def load_csv(self):
         try:
-            with open(self.csvfile, newline='') as c:
+            with open(self.csvfile, newline='', encoding="utf8") as c:
```

Comment:
Nice, I wonder how this would impact on when sending the emails ... but looks like  a good idea